config/default.yml

```
@@ -1006,10 +1006,8 @@ Layout/LineContinuationLeadingSpace:
                   Use trailing spaces instead of leading spaces in strings
                   broken over multiple lines (by a backslash).
   Enabled: pending
-  AutoCorrect: false
-  SafeAutoCorrect: false
   VersionAdded: '1.31'
-  VersionChanged: '1.32'
+  VersionChanged: '1.45'
```

Can you specify `'<<next>>'` instead?

```suggestion
  VersionChanged: '<<next>>'
```

---

Ah, sure. Done!